README.md

```diff
@@ -214,17 +224,25 @@ Close the GUI and reopen to confirm the path is correct.
 
 # DeepProfiler Project Setup:
 
-Deep Profiler must be installed via Github.
+DeepProfiler must be installed via Github.
 Commit [`2fb3ed3`](https://github.com/cytomining/DeepProfiler/commit/2fb3ed3027cded6676b7e409687322ef67491ec7) was used while developing `idrstream`.
 Install the repository into `idrstream/` with:
 ```sh
```

Comment:
```suggestion
```console
```

Reply:
changed in https://github.com/WayScience/IDR_stream/pull/12/commits/5cf59bb6be7873572cf196ebb2babb3f015f2bb7